lls/main.c

```diff
@@ -270,7 +413,25 @@ lls_stage2(void *arg)
 			return ret;
 	}
 
-	/* TODO Add ND filters. */
+	/* We receive ND packets on the front interface from the GK block. */
+
+	/*
+	 * TODO Have a different block set up RSS on the back interface,
```

Comment:
Yes, I think we will need to do this using the RT-like functionality, although I think we are merging the RT block into the GK block.